OMGVA PoS/Data layer/Models/Giftcard.cs

```diff
@@ -1,9 +1,12 @@
 ﻿namespace OMGVA_PoS.Data_layer.Models
 {
-    public class Giftcard
+    public class GiftCard
```

Comment:
Why did you change this? It is giftcard in the specification. Also, you did not change the file name.

Reply:
Since gift card is two words it should be GiftCard. Reverted now, however, done according to specification